Sources/Glob/GlobSearch.swift

```diff
@@ -79,6 +79,9 @@ public func search(
 						var options: FileManager.DirectoryEnumerationOptions = []
 						if skipHiddenFiles {
 							options.insert(.skipsHiddenFiles)
+                            #if !os(Linux)
+                            options.insert(.producesRelativePathURLs)
+                            #endif
```

Comment:
The strategy I chose doesn't work on Linux as these options are not available in the LInux version of the Foundation (thanks, Apple). Let me know what you think about the PR and how important it is to make Linux work for that scenario as well and I can try spend a bit more time into figuring this out.